setup.yml

```diff
@@ -0,0 +1,174 @@
+---
+- name: Setup Hush Line application server
+  hosts: all
+  become: yes
+  vars_files:
+    - vars/common.yml
+    - vars/secrets/secrets.yml
+  tasks:
+    - name: Install required packages
+      apt:
+        name:
+          - python3
+          - python3-pip
+          - git
+          - nginx
+          - default-mysql-server
+          - python3-venv
+          - certbot
+          - python3-certbot-nginx
+          - libnginx-mod-http-geoip
+          - ufw
+          - fail2ban
+          - redis
+          - redis-server
+          - apt-transport-https
+          - curl
+          - sqlite3
+          - acl
+        update_cache: yes
+        state: latest
+
+    - name: Clone Hush Line repository
```

Comment:
This shouldn't be a clone. That implies that we're taking the latest `HEAD` version of the default branch (currently `main`). All the files are local to the repo and should be copied over with `rsync` or something similar.